bin/alluxio

```diff
@@ -257,13 +257,8 @@ function main {
     runJavaClass "$@"
   ;;
   "runTest")
+    PARAMETER="configured" # instructs runTests to use the configured master address
```

Comment:
would it be possible to modify TestRunner to use flags (as opposed to assigning implicit meaning to command-line arguments)?